### PR TITLE
fix(ctb): Update universal contracts with Triple Slash Natspec Styling

### PR DESCRIPTION
**Pull request by refcell**
**Description**

Updates `contracts-bedrock/contracts/universal/*` contracts and associated tests
to the triple slash natspec comment style.

### CODE REVIEW

**Comment by refcell** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6096** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6096" target="_blank">Graphite</a> 
    * **PR #6099** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6099" target="_blank">Graphite</a> 
      * **PR #6100** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6100" target="_blank">Graphite</a> 
        * **PR #6125** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6125" target="_blank">Graphite</a> 
          * **PR #6126** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6126" target="_blank">Graphite</a> 
            * **PR #6128** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6128" target="_blank">Graphite</a> 
              * **PR #6130** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6130" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6130?utm_source=stack-comment).

**Comment by refcell** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6096** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6096" target="_blank">Graphite</a> 
    * **PR #6099** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6099" target="_blank">Graphite</a> 
      * **PR #6100** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6100" target="_blank">Graphite</a> 
        * **PR #6125** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6125" target="_blank">Graphite</a> 
          * **PR #6126** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6126" target="_blank">Graphite</a> 
            * **PR #6128** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6128" target="_blank">Graphite</a> 
              * **PR #6130** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6130" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6130?utm_source=stack-comment).

File: packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol [hunk 1]
```diff
@@ -470,50 +384,38 @@ abstract contract CrossDomainMessenger is
             RELAY_GAS_CHECK_BUFFER;
     }
 
-    /**
-     * @notice Intializer.
-     */
+    /// @notice Intializer.
```
Comment (clabby) [inline packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol:387]:
```suggestion
    /// @notice Initializer.
```

File: packages/contracts-bedrock/contracts/universal/FeeVault.sol [hunk 2]
```diff
@@ -5,73 +5,50 @@ import { L2StandardBridge } from "../L2/L2StandardBridge.sol";
...
+///         used to hold fee revenue generated by the L2 system.
 abstract contract FeeVault {
-    /**
-     * @notice Enum representing where the FeeVault withdraws funds to.
-     * @custom:value L1  FeeVault withdraws funds to L1.
-     * @custom:value L2  FeeVault withdraws funds to L2.
-     */
+    /// @notice Enum representing where the FeeVault withdraws funds to.
+    /// @custom:value L1  FeeVault withdraws funds to L1.
+    /// @custom:value L2  FeeVault withdraws funds to L2.
```
Comment (clabby) [inline packages/contracts-bedrock/contracts/universal/FeeVault.sol:14]:
```suggestion
    /// @custom:value L1 FeeVault withdraws funds to L1.
    /// @custom:value L2 FeeVault withdraws funds to L2.
```

**Review by clabby — APPROVED** [review]
LGTM <3

**Pull request merged**